tests/test_asm.rs

```diff
@@ -467,7 +467,7 @@ fn test_zero_address() {
 fn test_memoized_secp256k1() {
     let isa = ISA_IMC;
     let version = VERSION1;
-    let buffer = fs::read("benches/data/secp256k1_bench").unwrap().into();
+    let buffer = fs::read("tests/programs/secp256k1").unwrap().into();
```

Comment:
I was just thinking: do we still need secp256k1 as a test here? I do recall test suite already contains secp256k1 test, can we simply remove it in this repo for simplicity?

We could probably draw a line: in this repo, we only do small, instruction level testing. And we leave it to the test suite repo for more, contract level testing.

Any thoughts?

Reply:
I agree.

Currently, all the test programs are hand-written except for the secp256k1 test; we can clearly know which instruction the test program is testing.

This makes the secp256k1 test look strange here.